app/views/general/search.html.erb

```diff
@@ -202,6 +202,8 @@
     <div class="results_section">
       <% if @xapian_requests_hits == 1 && @page == 1 %>
         <h2 class="foi_results"><%= _("One FOI request found") %></h2>
+      <% elsif @xapian_requests.matches_estimated > @requests_per_page %>
+        <h2 class="foi_results"><%= _("FOI requests {{start_count}} to {{end_count}} of about {{total_count}}", :start_count => ((@page-1)*@requests_per_page+1).to_s, :end_count => [@page*@requests_per_page, @xapian_requests.matches_estimated].min.to_s, :total_count =>  @xapian_requests.matches_estimated.to_s) %></h2>
```

Comment:
Why the extra conditionals?

Can't we make this much simpler and just s/`{{total_count}}`/`about {{total_count}}`/?